src/utils.ts

```diff
@@ -15,16 +15,47 @@ export function arb_set(set: Set<any>) { //skipcq: JS-0323
     return e;
   }
 }
+
 export function allSolutions(
   assignment: t_assignment,
   allSolutions: Set<t_assignment>,
 ) {
   allSolutions.add(assignment);
 }
 
+export function prepare_constraints_for_eval(
+  variables: Set<string>,
+  constraints: Set<string>,
+) {
+  const new_constraints = new Set<string>();
+  constraints.forEach(function (constraint) {
+    variables.forEach(function (variable) {
+      const regex = new RegExp(
+        `(?<=[^\w^']|^)${variable}(?=[^\w^'^(]|$)`,
```

Comment:
Please adjust the regex so that it is irrelevant if ' or " is used in the string. Also in Line 56